dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java

```diff
@@ -244,7 +251,8 @@ public enum FileType {
         DOCKSTORE_YML(FileTypeCategory.OTHER),
         DOCKSTORE_SERVICE_YML(FileTypeCategory.PRIMARY_DESCRIPTOR), DOCKSTORE_SERVICE_TEST_JSON(FileTypeCategory.TEST_FILE), DOCKSTORE_SERVICE_OTHER(FileTypeCategory.OTHER),
         DOCKSTORE_GXFORMAT2(FileTypeCategory.GENERIC_DESCRIPTOR), GXFORMAT2_TEST_FILE(FileTypeCategory.TEST_FILE),
-        DOCKSTORE_SWL(FileTypeCategory.GENERIC_DESCRIPTOR), SWL_TEST_JSON(FileTypeCategory.TEST_FILE);
+        DOCKSTORE_SWL(FileTypeCategory.GENERIC_DESCRIPTOR), SWL_TEST_JSON(FileTypeCategory.TEST_FILE),
+        DOCKSTORE_IPYNB(FileTypeCategory.PRIMARY_DESCRIPTOR), DOCKSTORE_REES(FileTypeCategory.CONFIGURATION_FILE), DOCKSTORE_DATA(FileTypeCategory.OTHER);
```

Comment:
Nit, DOCKSTORE_DATA seems a little too generic to me; I'd scope it down to something more notebook specific, if it makes sense. IOW, I can tell what the other file types are used for by their name only, this one is a little vaguer. DOCKSTORE_NOTEBOOK_DATA? May not be precise, and maybe there's not a better alternative.

Reply:
Changed to `DOCKSTORE_NOTEBOOK_REES` and `DOCKSTORE_NOTEBOOK_OTHER`, respectively, the latter of which is consistent with the naming scheme in the `Service` implementation...